src/App.svelte

```diff
@@ -21,9 +21,9 @@
 	<main>
 		<nav>
 			<!-- Don't use Link to prevent code being executed on backgroud from previous page -->
```

Comment:
What about this comment?

Reply:
Deleted, now consensus is shared and there's no problem using Link